jquery.swiftype.autocomplete.js

```diff
@@ -325,8 +334,7 @@
         $this.cache.put(norm, data.records);
       } else {
         $this.addEmpty(norm);
-        $this.data('swiftype-list').empty();
```

Comment:
this doesn't seem to be getting called anymore - the call to `$list.empty();` inside `showNoResults` is acting on `var $list = $('<' + config.suggestionListType + ' />').appendTo($listContainer);`

Reply:
They should be the same object. `$list` gets added to the jquery data object in the next line:

```
$this.data('swiftype-list', $list);
```